File: App/Minority/Minority/app/src/main/java/com/revich/mobile/minority/VotoACalcular.java
package com.revich.mobile.minority;

/**
 * Created by Igal on 9/10/2017.
 */

public class VotoACalcular {
    public String OpcionA ;
    public String OpcionB ;
    public String VotoJugador ;
    public int IdUsuario ;
    public int IdSala ;
    public int NRonda ;

    public void LlenarDatos(int usuario, String opcionA, String opcionB, String votoJugador, int idSala, int nronda)
    {
        this.OpcionA=opcionA;
        this.OpcionB=opcionB;
        this.VotoJugador=votoJugador;
        this.IdUsuario=usuario;
        this.IdSala=idSala;
        this.NRonda=nronda;
    }
}

File: RestApi/RestApiMinority/RestApiMinority/Models/VotoACalcular.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace RestApiMinority.Models
{
    public class VotoACalcular
    {
        public string OpcionA { get; set; }
        public string OpcionB { get; set; }
        public string VotoJugador { get; set; }
        public int IdUsuario { get; set; }
        public int IdSala { get; set; }
        public int NRonda { get; set; }
    }
}